apps/nextjs-website/src/components/organisms/Auth/ResetPasswordForm.tsx

```diff
@@ -8,38 +8,49 @@ import {
   Card,
   Grid,
   Box,
+  TextField,
 } from '@mui/material';
 import { emailMatcher } from '@/helpers/auth.helpers';
-import { translations } from '@/_contents/translations';
-import RequiredTextField, {
-  ValidatorFunction,
-} from '@/components/molecules/RequiredTextField/RequiredTextField';
 import { IllusDataSecurity } from '@pagopa/mui-italia';
-import { Dispatch, SetStateAction, useEffect, useState } from 'react';
-
-const {
-  auth: { resetPassword },
-  shared,
-} = translations;
+import { Dispatch, SetStateAction, useCallback, useState } from 'react';
+import { useTranslations } from 'next-intl';
 
 interface ResetPasswordFormProps {
   email: string;
   setEmail: Dispatch<SetStateAction<string>>;
   handleResetPassword: () => Promise<void>;
-  emailValidators: ValidatorFunction[];
 }
 
 const ResetPasswordForm = ({
   email,
   setEmail,
   handleResetPassword,
-  emailValidators,
 }: ResetPasswordFormProps) => {
-  const [isSubmitDisabled, setSubmitDisabled] = useState(false);
+  const resetPassword = useTranslations('auth.resetPassword');
+  const shared = useTranslations('shared');
+
+  const [emailError, setEmailError] = useState<string | null>(null);
+
+  const validateForm = useCallback(() => {
+    const emailError =
+      !email || email?.trim().length === 0
+        ? shared('requiredFieldError')
+        : !emailMatcher.test(email)
+        ? shared('emailFieldError')
+        : null;
```

Comment:
This logic is duplicate in apps/nextjs-website/src/components/organisms/Auth/LoginForm.tsx, maybe to put it in a util of some sort?